addon/components/layers/wms-layer.js

```diff
@@ -96,7 +58,7 @@ export default TileLayer.extend({
     Leaflet layer or promise returning such layer.
   */
   createLayer() {
-    return L.tileLayer.wms(this.get('url'), this.get('options'));
+    return new L.TileLayer.WMS(this.get('url'), this.get('options'));
```

Comment:
you could left old one propagation. L.tileLayer.wms is a factory method of L.TileLayer.WMS